pkg/tester/data.go

```diff
@@ -173,6 +175,25 @@ func InitializeData(
 	blockStorage := storage.NewBlockStorage(localStore)
 	balanceStorage := storage.NewBalanceStorage(localStore)
 
+	// Bootstrap balances, if provided. We need to do before initializing
+	// the reconciler otherwise we won't reconcile bootstrapped accounts
+	// until rosetta-cli restart.
+	if len(config.Data.BootstrapBalances) > 0 {
+		_, err := blockStorage.GetHeadBlockIdentifier(ctx)
+		if err == storage.ErrHeadBlockNotFound {
+			err = balanceStorage.BootstrapBalances(
+				ctx,
+				config.Data.BootstrapBalances,
+				genesisBlock,
+			)
+			if err != nil {
+				log.Fatalf("%s: unable to bootstrap balances", err.Error())
+			}
+		} else {
+			log.Println("Skipping balance bootstrapping because already started syncing")
+		}
```

Comment:
Do we want to handle the case where `err != nil && err != torage.ErrHeadBlockNotFound` ? Or maybe log it? IMO this is now highly coupled with the error that `blockStorage.GetHeadBlockIdentifier` returns.

Reply:
This code has been sitting here for a while...only green because I moved it above reconciler initialization. I'll log it!